build_handwritten.yaml

```diff
@@ -253,6 +253,7 @@ python_dependencies:
   - address_sorting
   - ares
   - boringssl
+  - re2
```

Comment:
this looks legit. It's a list of libraries that should be compiled into python artifacts.